alex/applications/PublicTransportInfoEN/directions.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
```

Comment:
Wouldn't it be nicer to have just one class for CS & EN Google directions when it's the same API?